.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ubuntu-20.04
```

Comment:
we should instead specify `{{ matrix.os }}` here and define OS in matrix section.